features/sharding/distsql/handler/src/main/java/org/apache/shardingsphere/sharding/distsql/handler/checker/ShardingTableRuleStatementChecker.java

```diff
@@ -122,6 +123,31 @@ public static boolean isValidBindingTableGroups(final Collection<ShardingTableRe
         return check(toBeCheckedRuleConfig, dataSourceNames);
     }
     
+    private static Collection<String> getRequiredDataSource(final ShardingRuleConfiguration config) {
```

Comment:
Since the result is a collection, the method name should be plural: `getRequiredDataSources`

Reply:
Fixed, THX